lib/plugins/aws/info/display.js

```diff
@@ -60,11 +60,10 @@ module.exports = {
 
     if (info.functions && info.functions.length > 0) {
       info.functions.forEach((f) => {
-        functionsMessage += `\n  ${f.name}:\n`;
-        functionsMessage += `    ${f.arn}`;
+        functionsMessage = functionsMessage.concat(`\n  ${f.name}: ${f.arn}`);
```

Comment:
Just one quick question. Why using concat?

We should IMHO use one way of concatenating and use it throughout the whole info codebase.